Homeworks/homework_16/MyListener.py

```diff
@@ -0,0 +1,26 @@
+from robot.libraries.BuiltIn import BuiltIn
+
+class MyListener:
```

Comment:
Listener is expected to have end_keyword method that checks keywords' tags and makes screenshot if there is a 'screenshot' tag in the list of tags, other listener methods are not needed in this case

Reply:
MyListener has been updated
[Tags]    Screenshot has been added